testing/regression/cypress/e2e/features/page-library-sort.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User can view existing page library data here.
+
+    Background:
+        Given I am logged in as "superuser" and password ""
+
+    Scenario: User list Page name in descending and ascending order
+        Given User navigates to Page Library
+        When User views the Page library
+        And User click the up or down arrow in the Page name column
+        Then Page names are listed in descending order
+        When User click the page name up or down arrow again
```

Comment:
Suggestion: Instead of creating 2 step definitions that do the exact same thing (clicking the page name column sort), you could just re-use the one on line  9.

```cucumber
And User click the up or down arrow in the Page name column
Then Page names are listed in descending order
And User click the up or down arrow in the Page name column
```